cg/apps/housekeeper/hk.py

```diff
@@ -96,6 +96,24 @@ def files(
         """ Fetch files """
         return self._store.files(bundle=bundle, tags=tags, version=version, path=path)
 
+    @staticmethod
+    def fetch_file_from_version(
```

Comment:
could you just reuse this? 
https://github.com/Clinical-Genomics/housekeeper/blob/8238a1c95059c55aadc77ecd872b815778834b01/housekeeper/store/api/find.py#L68

Reply:
The reason why I did not was because I already have the version object so there is no need to request from the database over and over again